crates/subspace-farmer/src/plotting.rs

```diff
@@ -85,7 +85,7 @@ impl Drop for Plotting {
         // we don't have to do anything in here
         // these are for clarity and verbosity
         let _ = self.sender.take().unwrap().send(());
-        info!("Plotting stopped!");
+        // info!("Plotting stopped!");
```

Comment:
Why?

Reply:
The same message was also present in the racing, on `receiver.recv()`. So it was printing it double. I commented it, since we may want to use it in the future for debugging. But not necessary atm.

Reply:
Maybe remove it then?

Reply:
I liked the idea of having a print statement there as a comment (to emphasize we can debug it further). But I can remove it for sure